src/layer/text.js

```diff
@@ -293,6 +293,8 @@ var Text = function(parentEl) {
                 lineEl.className = "ace_line_group";
             } else {
                 lineEl.className = "ace_line";
+                lineEl.id = "ace_line_" + row;
```

Comment:
Yes, right. I didn't face any issues probably because I am setting another id (aria-id) to the selected line to be able to point to this line as an active descendant from the active element. 

What I could do is number each editor  (as you suggested), specify each line's id with which editor it belongs and its row number.